starky/src/permutation.rs

```diff
@@ -49,40 +56,21 @@ pub(crate) fn compute_permutation_z_polys<F, C, S, const D: usize>(
     config: &StarkConfig,
     challenger: &mut Challenger<F, C::Hasher>,
```

Comment:
It seems `challenger` is no longer needed since `get_n_permutation_challenge_sets` was moved outside of this method